test/unittest/bamboo_test.go

```diff
@@ -76,7 +77,7 @@ var BambooCorrectVariables = map[string]interface{}{
 	"dataset_url":          nil,
 	"installation_timeout": testTimeout,
 	"bamboo_configuration": map[string]interface{}{
-		"helm_version": "1.0.0",
+		"helm_version": "",
```

Comment:
Pass no version to make sure that in the helm_release resource the version is not null (Helm provider will automatically get the latest)